Tools/Scripts/webkitpy/__init__.py

```diff
@@ -35,7 +35,7 @@
 AutoInstall.set_directory(os.path.join(libraries, 'autoinstalled', 'python-{}-{}'.format(sys.version_info[0], platform.machine())))
 
 if sys.version_info >= (3, 7):
-    AutoInstall.register(Package('pylint', Version(2, 6, 0)))
+    AutoInstall.register(Package('pylint', Version(2, 13, 9)))
```

Comment:
This may not suffice:

```
pylint 2.13.9 requires astroid, which is not installed.
pylint 2.13.9 requires dill, which is not installed.
pylint 2.13.9 requires isort, which is not installed.
pylint 2.13.9 requires mccabe, which is not installed.
pylint 2.13.9 requires platformdirs, which is not installed.
```

Reply:
These are all used only by optional checkers, so let's not pull in all of this now.